### PR TITLE
fix: removed unnecessary api call from events page

### LINKED ISSUE #1095: [🐞 BUG] Remove loading problem in events page
### Description

After some time, the loading appears on the events page; it should be removed or commented out.

Please also add Hacktoberfest label

### Screenshots

image

### Browser 🥦

Brave

### Checklist ✅

- [X] I checked and didn't find similar [issue](https://github.com/MilanCommunity/Milan/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/MilanCommunity/Milan/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no).

### PR DESCRIPTION
**Pull request by AbhiPatel10**
closes #1095 

### 👷🏻 Changes made  

I have comment some code in events page which was not currently in use and also create unnecessary api calls which resolve the Loading problem. However i just comment the code if in the future the requirement will occur then we can use that code.

### CODE REVIEW

File: src/pages/Event/AllEvents/Events.jsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 import Footer from "../../../components/Footer/Footer";
 import Navbar from "../../../components/Navbar/Navbar";
 import ComingSoon from "../../../components/Cards/ComingSoon/ComingSoon";
-import { eventEndpoints } from "../../../assets/data/ApiEndpoints";
 import ComponentHelmet from "../../../utils/ComponentHelmet";
+// import Loading from "../../../components/Loading";
+// import SingleClubEvent from "../../../components/Cards/SingleClubEvent/SingleClubEvent";
+// import useSWR from "swr";
+// import fetcher from "../../../utils/Fetcher";
+// import { eventEndpoints } from "../../../assets/data/ApiEndpoints";
```
Comment (tamalCodes) [inline src/pages/Event/AllEvents/Events.jsx:10]: Can you please remove the code from here instead of commenting it out?

**Review by tamalCodes — CHANGES_REQUESTED** [review]

File: src/pages/Event/AllEvents/Events.jsx (hunk 1 above)
Comment (AbhiPatel10) [reply src/pages/Event/AllEvents/Events.jsx:10]: Sure

**Comment by AbhiPatel10** [comment]
@tamalCodes  I have removed Comment code please have a look

**Review by tamalCodes — APPROVED** [review]
# Amazing work 🎉
**The changes look good to me and will be merged soon.**

Do follow [Tamal](https://github.com/tamalCodes) for more Opensource fun projects and don't forget to drop a star so that you get updated about our latest [releases](https://github.com/MilanCommunity/Milan/releases) (we will tag you and mention your work) and also a [shoutout](https://www.linkedin.com/feed/update/urn:li:activity:7073574804519354368/) on social media (LinkedIn and Twitter) !

Happy Opensource 🚀.

**Pull request merged**